index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Favourite Meal</title>
+</head>
+<body>
+    <h1>Healthy Ramen</h1>
+    <br>
```

Comment:
Using a `<br>` tag is totally ok at this point. However as you get better at CSS/HTML you will want to do this in stylesheets.

This would achieve something similar
```css
h1 {
  padding-bottom: 0.5em
}
```